### PR TITLE
feat: support include directive in GitHub issues / PRs / code blocks

### PR DESCRIPTION
**Pull request by kumachan-mis**
example https://github.com/WillBooster/plantuml-visualizer/blob/master/puml-sample/state.pu

### CODE REVIEW

**Comment by kumachan-mis** [comment]
Include directive in PR would be already supported.
I will test

File: src/mutator/DescriptionMutator.ts [hunk 1]
```diff
@@ -3,25 +3,27 @@ import { markAsAlreadyProcessed, textToImage } from './MutatorUtil';
 
 export const DescriptionMutator = {
   async embedPlantUmlImages(finders: Finder[], webPageUrl: string, $root: JQuery<Node>) {
-    for (const finder of finders) {
-      for (const content of finder.find(webPageUrl, $root)) {
-        const $text = content.$text;
+    const arraysOfContents = await Promise.all(finders.map((finder) => finder.find(webPageUrl, $root)));
```
Comment (exKAZUu) [inline src/mutator/DescriptionMutator.ts:6]: This statement waits all the promises, so I feel it is not good performance-wise.
Could we rewrite this as follows?

```
await Promise.all(finders.map(async (finder) => {
  const contents = await finder.find(webPageUrl, $root);
  for (const content of contents) {
    .. snip ..
  }
}));
```

Comment (kumachan-mis) [reply src/mutator/DescriptionMutator.ts:6]: Shall I rewrite DiffMutator, too?

Comment (exKAZUu) [reply src/mutator/DescriptionMutator.ts:6]: yes, please

**Pull request merged**